integrationtest/src/main/java/software/amazon/timestream/integrationtest/DatabaseMetaDataOneDBNoTBIntegrationTest.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright <2020> Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.timestream.integrationtest;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import software.amazon.timestream.jdbc.TimestreamDatabaseMetaData;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.SQLException;
+
+/**
+ * Integration tests for supported getters in {@link TimestreamDatabaseMetaData}
+ */
+class DatabaseMetaDataOneDBNoTBIntegrationTest {
+  private DatabaseMetaData metaData;
```

Comment:
I think these two variables could be deleted for each test.

Reply:
let me remove them